src/trap.cpp

```diff
@@ -320,9 +320,6 @@ bool trap::can_see( const tripoint &pos, const Character &p ) const
 
 void trap::trigger( const tripoint &pos ) const
 {
-    if( is_null() ) {
```

Comment:
This seems like it could be a problem to remove. What was happening, is it considered Null if it's triggered by a sound with no source?

Reply:
Didn't mean to remove this. I originally made that trigger overload call another trigger overload with the null check, but reverted after without readding it. Will add again.